cmd/run.go

```diff
@@ -75,7 +75,7 @@ var runCmd = &cobra.Command{
 		var localCloud *cloud.LocalCloud
 		go func() {
 			// Start the local cloud service analogues
-			localCloud, err = cloud.New(proj.Name)
+			localCloud, err = cloud.New(proj.Name, cloud.LocalCloudOptions{})
```

Comment:
Are we also planning on adding SSL support for `run`

Reply:
How can I help test this baddy, would be awesome to have this merged as I could use this feature for my project at work, any ETA on when this might be live on prod?

Reply:
@algoflows just pushed changes to also implement this for `nitric run`, so should be released shortly.